website/docs/docs/introduction.md

```diff
@@ -3,188 +3,65 @@ title: "What is dbt?"
+- No longer copy and paste SQL, which can lead to errors when logic changes. Instead, build reusable data models that get pulled into subsequent models and analysis. Change a model once and that change will propagate to all its dependencies.
+- Publish the canonical version of a particular data model, encapsulating all complex business logic. All analysis on top of this model will incorporate the same business logic without needing to reimplement it.
+- Use mature source control processes like branching, pull requests, and code reviews.
+- Write data quality tests quickly and easily on the underlying data. Many analytic errors are caused by edge cases in the data: testing helps analysts find and handle those edge cases.
 
-:::info Check out our sample dbt project
+## dbt products
 
-Want to check out a sample project? Have a look at our [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop) project on GitHub!
+You can access dbt using dbt Core or dbt Cloud. dbt Cloud is built around dbt Core, but it also provides:
 
-:::
+- Web-based UI so it’s more accessible
+- Hosted environment so it’s faster to get up and running
+- Differentiated features, such as metadata, in-app job scheduler, observability, integrations with other tools, integrated development environment (IDE), and more.
 
-### Database Connections
-dbt connects to your <Term id="data-warehouse" /> to run data transformation queries. As such, you’ll need a data warehouse with source data loaded in it to use dbt. dbt natively supports connections to Snowflake, BigQuery, Redshift and Postgres data warehouses, and there’s a number of community-supported adapters for other warehouses (see [Supported Data Platforms](supported-data-platforms)).
+You can learn about plans and pricing on [www.getdbt.com]([https://www.getdbt.com/pricing/](https://www.getdbt.com/pricing/)).
 
-When you define your connection, you’ll also be able to specify the target schema where dbt should create your models as tables and <Term id="view">views</Term>. See [Managing environments](managing-environments)  for more information on picking target schema names.
+### dbt Cloud
 
+dbt Cloud is the fastest and most reliable way to deploy dbt. Develop, test, schedule, and investigate data models all in one web-based UI. Read more about [Getting started with dbt Cloud](/docs/get-started/getting-started/set-up-dbt-cloud) and [dbt Cloud features](/docs/get-started/dbt-cloud-features).
 
-### dbt Commands
-A command is an instruction to execute dbt which compiles and runs SQL against the database.
+### dbt Core
 
-When you issue a dbt command, such as `run`, dbt:
+dbt Core is an open-source tool that enables data teams to transform data using analytics engineering best practices. You can install and use dbt Core on the command line. Read more about [Getting started with dbt Core](/docs/get-started/getting-started-dbt-core).
```

Comment:
I have zero issues with the colloquial "on the command line" and use it all the time myself but the official guidance is "Customers enter commands at a command prompt, not on a command line" or use a "command line tool."  I don't know if that's something we want to standardize on.

Reply:
Let's do a cleanup of this after Coalesce.